openwisp_controller/config/admin.py

```diff
@@ -661,12 +663,16 @@ def create_log_entry(user, clone):
                 pk__in=user.organizations_dict.keys()
             )
         if selectable_orgs:
-            if request.POST.get('organization'):
+            organization = request.POST.get('organization')
+            if organization or organization == '':
```

Comment:
this if looks redundant with the new if added after, I would avoid it

Reply:
If the organization is system shared then organization value here is `''`, that's why we have to add this condition else it will not save ever for system shared org.